files/nginx.conf

```diff
@@ -86,13 +89,15 @@ http {
       # Image processing for images in local file
       location ~ ^/local/small_light[^/]*/(.+)$ {
         set $small_light_maximum_size 3072;
-        perl validator::handler;
+        mruby_content_handler /opt/nginx/ruby/lib/validator.rb;
+	error_page 400 = @error_400;
       }
 
       # Image processing for images in AWS S3
       location ~ ^/small_light[^/]*/(.+)$ {
         set $small_light_maximum_size 3072;
-        perl validator::handler;
+        mruby_content_handler /opt/nginx/ruby/lib/validator.rb;
+	error_page 400 = @error_400;
```

Comment:
Fixed in https://github.com/wantedly/nginx-image-server/pull/44/commits/6af385f5e4a0b9b914fe04644ff66f5b0e5f4f65.